lambeq/backend/drawing/drawable.py

```diff
@@ -95,7 +95,7 @@ class WireEndpoint:
 
     x: float
     y: float
-
+    color: int = 0 # New attribute for wire noun
```

Comment:
The wire endpoint shouldn't know that the `noun_id` will be used for color here. Kindly rename it to something like `noun_id`. It's the drawing backend that tries to map the noun IDs to colors.